cypress/e2e/products/productsVariants.js

```diff
@@ -79,32 +79,52 @@ describe("As an admin I should be able to create variant", () => {
             productId: createdProduct.id,
             channelId: newChannel.id,
           });
-          cy.visit(`${urlList.products}${createdProduct.id}`);
+          cy.visit(`${urlList.products}${createdProduct.id}`)
+            .waitForProgressBarToNotBeVisible()
+            .get(PRODUCT_DETAILS.addVariantButton)
+            .should("exist")
+            .click()
+            .get(PRODUCT_DETAILS.dataGridTable)
+            .should("be.visible")
+            .get(PRODUCT_DETAILS.firstRowDataGrid)
+            .click({ force: true })
+            .type(name)
+            .get(BUTTON_SELECTORS.confirm)
+            .click()
+            .confirmationMessageShouldAppear()
+            .reload()
+            .waitForProgressBarToNotBeVisible()
+            .get(PRODUCT_DETAILS.dataGridTable)
+            .should("be.visible")
+            .wait(1000)
+            .get(BUTTON_SELECTORS.showMoreButton)
+            .click()
+            .get(PRODUCT_DETAILS.editVariant)
+            .click()
+            .get(VARIANTS_SELECTORS.manageChannels)
+            .click()
+            .get(VARIANTS_SELECTORS.allChannels)
+            .check()
+            .get(BUTTON_SELECTORS.submit)
+            .click();
```

Comment:
Can we split it to few functions? For eg. addVariantToDatagrid, changeVarinatNameInDatagrid, enterVariantEditPage, selectChannelsForVariant?

Reply:
If it is not a problem we could do this in another PR, ok? :)